### ISSUE TITLE
[Bug]: docs mention .env_docker.example but it no longer exists in repo

### ISSUE THREAD

**Issue by troyfolger** [Sygil-Dev/sygil-webui#1759]
### What happened?

A bug happened!

commit c4fa1101da41c1f3c8f454eda3a43f82f16c71d1
Author: hlky <106811348+hlky@users.noreply.github.com>
Date:   Tue Oct 4 12:58:30 2022 +0100

    Delete .env_docker.example

... but the docs still suggest it is mandatory for running the docker image.

### Version

0.0.1 (Default)

### What browsers are you seeing the problem on?

Chrome

### Where are you running the webui?

Linux

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by troyfolger** [comment]
hmmm guess I'm missing something. There seems to be a big mismatch in the documentation, and the git actions on October 4?

```
$ git show 0725e920f3e84e9cb14c5f8fe3d2f037f911c4dd --summary
commit 0725e920f3e84e9cb14c5f8fe3d2f037f911c4dd
Author: hlky <106811348+hlky@users.noreply.github.com>
Date:   Tue Oct 4 12:52:11 2022 +0100

    remove old docker files

 delete mode 100644 Dockerfile
 delete mode 100755 build_docker.sh
 delete mode 100644 docker-compose.amd.yml
 delete mode 100644 docker-compose.override.yml
 delete mode 100644 docker-compose.yml
 delete mode 100755 docker-reset.sh
 delete mode 100755 entrypoint.sh
```

The docker compose approach has been removed? 

Here is what the docs have:
https://github.com/Sygil-Dev/sygil-webui/blame/master/docs/1.Installation/4.docker-guide.md#L102

**Comment by sammcj** [comment]
This was pretty confusing! I'm not sure where the docker-compose examples are now?

**Comment by ZeroCool940711** [comment]
Hey guys, sorry about this issue, I will be adding the `.env_docker.example` on the root of the repo, I found an old PR from when that file was made (#730) so I will just copy the same content from it and make the file again. Give me a couple of minutes to make a PR with the changes.

**Comment by ZeroCool940711** [comment]
Should be fixed with PR #1818, the `.env_docker.example` file is on the root of the repo again, let me know if there are any other issues with running or building the docker image so I can fix them.
